### PR TITLE
[V3] Poetry Club Door Policy: fix everything around this exercise

### PR DESCRIPTION
**Pull request by SleeplessByte**
# Reviewers, please spell/grammar check

Related #1017 

## Changes

- Rename from concept based exercise (`strings`) to story based exercise `policy-club-door-policy`
- Add full name to `config.json`
- Add blurb for `strings` to `config.json`
- Add a small welcome block to the stub
- Add design document
- Sync concept documents with exercise documents
- Add concept links
- Add about.md for strings concept

### CODE REVIEW

File: exercises/concept/poetry-club-door-policy/.meta/exemplar.js [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
+// 👋🏽 Hi again!
+//
+// A quick reminder about exercise stubs:
+//
+// 💡 You're allowed to completely clear any stub before you get started. Often
+// we recommend using the stub, because they are already set-up correctly to
+// work with the tests, which you can find in ./freelancer-rates.spec.js.
+//
+// 💡 You don't need to write JSDoc comment blocks yourself; it is not expected
+// in idiomatic JavaScript, but some companies and style-guides do enforce them.
```
Comment (kotp) [inline exercises/concept/poetry-club-door-policy/.meta/exemplar.js:18]: We might remove the "it is not expected in idiomatic JavaScript" if it is also not expected in idiomatic JavaScript that they are not used.

```javascript
// 💡 While you may write JSDoc comment blocks yourself it is not expected,
//  but be aware that some companies and style-guides do enforce them.
```

File: exercises/concept/poetry-club-door-policy/door-policy.js [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
+// 👋🏽 Hi again!
+//
+// A quick reminder about exercise stubs:
+//
+// 💡 You're allowed to completely clear any stub before you get started. Often
+// we recommend using the stub, because they are already set-up correctly to
+// work with the tests, which you can find in ./freelancer-rates.spec.js.
+//
+// 💡 You don't need to write JSDoc comment blocks yourself; it is not expected
+// in idiomatic JavaScript, but some companies and style-guides do enforce them.
```
Comment (kotp) [inline exercises/concept/poetry-club-door-policy/door-policy.js:18]: Same comment as with this section in `exemplar.js`.

**Review by ErikSchierboom — APPROVED** [review]

File: concepts/strings/about.md [hunk 3]
```diff
@@ -27,6 +27,24 @@ Some of the most-used operations on strings are to check their `length` and to c
 // => "I like cats"
 ```
 
-You can find all the methods in the [MDN docs][mdn docs].
+You can find all the methods in the [MDN docs][mdn-docs].
 
-[mdn docs]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String#Instance_methods
+## String interpolation
+
+Strings in JavaScript support interpolation using [string template literals][mdn-template-strings].
```
[the remark this answers is not in the document]
Comment (SleeplessByte) [reply concepts/strings/about.md:34]: Oh yeah smart!

File: exercises/concept/poetry-club-door-policy/.docs/hints.md [hunk 4]
```diff
@@ -2,34 +2,26 @@
...
+- Capitalization means having a single Uppercase character, followed by lowercase characters.
+- There are two methods to take a part of a string (for example, everything except the first character).
 - `.slice(position)` can slice up a string at a position.
 
 ## 3. Get the last letter of a sentence
 
-- This is similar to getting the first letter, except that it might be
-  preferable to use the number of characters in the string to find the position
+- This is similar to getting the first letter, except that it might be preferable to use the number of characters in the string to find the position
   of the last character.
```
[the remark this answers is not in the document]
Comment (SleeplessByte) [reply exercises/concept/poetry-club-door-policy/.docs/hints.md:17]: Nope, you are right! Thanks 🗡️

File: exercises/concept/poetry-club-door-policy/.docs/hints.md [hunk 5]
```diff
@@ -2,34 +2,26 @@
...
 - `.charAt(position)` will get the character at `position`.
 
 ## 2. Capitalize a word
 
-- Capitalization means having a single Uppercase character, followed by
-  lowercase characters.
-- There are two methods to take a part of a string (for example, everything
-  except the first character).
+- Capitalization means having a single Uppercase character, followed by lowercase characters.
+- There are two methods to take a part of a string (for example, everything except the first character).
```
Comment (SleeplessByte) [inline exercises/concept/poetry-club-door-policy/.docs/hints.md:11]:
```suggestion
- There are two methods to take a portion of a string (for example, everything except the first character).
```

**Pull request merged**